website/setupTypedoc.ts

```diff
@@ -1,22 +1,23 @@
-const fs = require("fs");
-const path = require("path");
+import fs from "fs";
+import path from "path";
 
 // Define the directory where the Typedoc HTML files are located
-const typedocDir = path.join(__dirname, "static/typedoc_output");
+const typedocDir = path.resolve(__dirname, "static/typedoc_output");
 
 // Define a recursive function to find all HTML files in a directory
 function findHtmlFiles(dir: string, fileList: string[] = []) {
   const files = fs.readdirSync(dir);
+  let list: string[] = [];
```

Comment:
Looks like this script is now only returning the html files inside the last directory it recurses into. Please could you amend this? one possible fix would be:

```ts
// Define a recursive function to find all HTML files in a directory
function findHtmlFiles(dir: string) {
  const files = fs.readdirSync(dir);
  let list: string[] = [];

  files.forEach((file: string) => {
    if (fs.statSync(path.join(dir, file)).isDirectory()) {
      list.concat(findHtmlFiles(path.join(dir, file)));
    } else if (file.endsWith(".html")) {
      list.push(path.join(dir, file));
    }
  });

  return list;
}
```